galleries/users_explain/text/fonts.py

```diff
@@ -185,7 +185,7 @@
    fig, ax = plt.subplots()
    ax.text(
        .5, .5, "There are 几个汉字 in between!",
-       family=['DejaVu Sans', 'WenQuanYi Zen Hei'],
+       family=['DejaVu Sans', 'Noto Sans CJK'],
```

Comment:
This failed to build:
```suggestion
       family=['DejaVu Sans', 'Noto Sans CJK JP'],
```